### ISSUE TITLE
Integration of models on the frontend and download music feature

### ISSUE THREAD

**Issue by ParamThakkar123** [AryanNanda17/NonStop-Lofi-Vibe-Generator#47]

**Comment by AryanNanda17** [comment]
In Nonstop Music (rated music button), the generated music isn't playing on the site. The contributor needs to ensure the generated music is properly sent to the site and starts playing. Additionally, there should be controls like pause, play, and options to skip forward by 10 seconds, download music etc.

**Comment by AryanNanda17** [comment]
We can make a GSoC project out of this issue. Seems large. :)

**Comment by ParamThakkar123** [comment]
> We can make a GSoC project out of this issue. Seems large. :)

Yes actually :sweat_smile: . It's a bit larger one compared to the others. might require some effort and programming experience with the frameworks used

**Comment by anuj846k** [comment]
> In Nonstop Music (rated music button), the generated music isn't playing on the site. The contributor needs to ensure the generated music is properly sent to the site and starts playing. Additionally, there should be controls like pause, play, and options to skip forward by 10 seconds, download music etc.


i can work on this

**Comment by ParamThakkar123** [comment]
Thanks for your interest @anuj846k but this project is slightly large compared to the previous ones. Our recommendation would be to complete the ones already assigned to you first.

**Comment by ParamThakkar123** [comment]
We would assign this to you as soon as you solve the issues currently assigned to you.
